pkgs/applications/audio/renoise/beta.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv, lib, requireFile, fetchurl, libX11, libXext, libXcursor, 
+  libXrandr, libjack2, alsaLib, personal_sha256, 
+  ... }:
+
+stdenv.mkDerivation rec {
+  name = "renoise_" + version ;
+  version = "beta_4";
+
+  buildInputs = [ libX11 libXext libXcursor libXrandr alsaLib libjack2 ];
+
+  src = let name = if builtins.currentSystem == "x86_64-linux" then
+                     "rns_3_1_0b4_linux_x86_64.tar.gz"
+                   else 
+                     "rns_3_1_0b4_linux_x86.tar.gz";
+            message = ''
+              please go to http://backstage.renoise.com/frontend/app/index.html#/login and download the binary.
+              import the binary by : nix-store --add-fixed sha256 ${name}
+              find out the hash by : shasum -a 256 ${name}
+              and set the hash in your /etc/nixos/configuration.nix or ~/.nixpkgs/config.nix
+              systemPackages = [ pkgs.renoise.override { personal_sha256 = "the_sum" } ]
+              '';
+        in 
+          requireFile {
+            name = name;
+            sha256 = personal_sha256;
```

Comment:
Well that is not possible, because the hash is always different, because the tar is created for YOU personally when you download the program from backstage. This is why it is so complicated made.